i18n-embed/src/lib.rs

```diff
@@ -249,8 +249,9 @@ use log::{debug, error, info};
 use rust_embed::RustEmbed;
 use thiserror::Error;
 
-pub use gettext;
-pub use tr;
+#[cfg(feature = "gettext-system")]
+pub use ::{gettext, tr};
```

Comment:
first time I've seen that syntax, but gets the job done I guess :+1:

Reply:
Heh, yeah import groups aren't usually used like that but I thought it would be nice not to have to repeat the cfg attribute :slightly_smiling_face:

Reply:
it's a nice trick not to repeat the cfg you're right! I'll try to get a release out today

Reply:
Okay, I released it in `v0.6.1` :slightly_smiling_face: thanks for the help